app/views/maintenance_tasks/tasks/index.html.erb

```diff
@@ -8,17 +8,6 @@
       </p>
     </div>
   <% else %>
-    <% if active_tasks = @available_tasks[:active] %>
-      <h3 class="title is-3">Active Tasks</h3>
-      <%= render partial: 'task', collection: active_tasks %>
-    <% end %>
-    <% if new_tasks = @available_tasks[:new] %>
-      <h3 class="title is-3">New Tasks</h3>
-      <%= render partial: 'task', collection: new_tasks %>
-    <% end %>
-    <% if completed_tasks = @available_tasks[:completed] %>
-      <h3 class="title is-3">Completed Tasks</h3>
-      <%= render partial: 'task', collection: completed_tasks %>
-    <% end %>
+    <%= render partial: 'results' %>
```

Comment:
If we can reduce the search and index routes into a single route then this change is probably not needed.